CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Grafana Mimir
 
 * [CHANGE] Flag `-azure.msi-resource` is now ignored, and will be removed in Mimir 2.7. This setting is now made automatically by Azure. #2682
+* [ENHANCEMENT] Added `<prefix>.tls-min-version` and `<prefix>.tls-cipher-suites` flags to configure cipher suites and min TLS version supported by servers. #2898
```

Comment:
What prefixes are available? Diff in help file shows only "server", nothing else.

Reply:
You're right.  This may be a confusion with the client-side flags.

Reply:
PR to clarify this: https://github.com/grafana/mimir/pull/3370